pihole_adlist_tool

```diff
@@ -75,6 +76,23 @@ declare -a all_domains
 CURRENT_DOMAIN=
 REGEX_ID=
 
+# version message
+print_version() {
+    echo "Pihole Adlist Tool v${PIHOLE_ADLIST_TOOL_VERSION}"
+}
+
+# fetches latest version
+fetch_remote_version() {
+	remote_version="$(
+		curl --silent https://api.github.com/repos/yubiuser/pihole_adlist_tool/releases/latest | 
+		grep '"tag_name":' | 
+		sed -E 's/.*"([^"]+)".*/\1/'
```

Comment:
```suggestion
		awk -F \" '{print $4}'
```

Personal preference, mostly because I've a hard time to understand more "complex" seds and tend to forget their interpretation fast